frontend/src/lib/components/PropertyFilters/Filter/TabbedPropertyFilter.tsx

```diff
@@ -1,16 +1,32 @@
 import React, { useState } from 'react'
 import { Col, Row, Select, Tabs } from 'antd'
 import { keyMapping } from 'lib/components/PropertyKeyInfo'
-import { cohortsModel } from '../../../models/cohortsModel'
+import { cohortsModel } from '../../../../models/cohortsModel'
```

Comment:
I think if you're going more than one level up, it would be better to use absolute paths.